docs/modify-cluster.md

```diff
@@ -13,17 +13,82 @@ You may need to perform any of the following administrative tasks to create or m
 
 If you need to create a machine pool, use the `create_machine_pool` [Terraform module](../examples/create_machine_pool/README.md).
 
+1. Export the number of replicas for your machine pool:
+    ```
+    export TF_VAR_replica=<integer>
+    ```
+1. Export the ID of the cluster where you want to create a machine pool. This ID can be found with the `rosa list cluster` command:
+    ```
+    export TF_VAR_cluster_id=<cluster_ID_string>
+    ```
+1. Export the name of your new machine pool:
+    ```
+    export TF_VAR_name=<name_of_machine_pool>
+    ```
+1. Export the machine type of your machine pool:
+    ```
+    export TF_VAR_machine_type=<machine_type>
+    ```
+1. **Optional**: Export a label by using a key-value pair with commas separating each label
+    ````
+    export TF_VAR_labels='{ test="first-label" }'
+1. Run `terraform apply` to update the machine pools.
+    ```
+    terraform apply
+    ```
+1. You see your machine pools are updated.
```

Comment:
I would just change these sentences to "Your machine pools should be updated" instead of the "you see" part. This applies to all instances of this on this page.